ultraLatency.py

```diff
@@ -182,6 +190,7 @@
         pass
     configureScanModule(ohboard, options.gtx, mode, mask,
                         scanmin=LATENCY_MIN, scanmax=LATENCY_MAX,
+                        stepsize=step,
```

Comment:
Yes, but we want the value of stepsize to be at max equal to scanmax-scanmin. So we assign to a variable the value of options.stepsize, rather than modify options.stepsize directly as the latter is a command line entry and could be required elsewhere as Brian pointed out. 

Therefore in the function, the variable with the updated value should be used, not whats input in the command line.